scripts/package/package_android_sdk.sh

```diff
@@ -84,20 +84,20 @@ echo "cd ${ANDROID_SDK_INSTALLDIR}"
 # e.g. export PATH=/usr/lib/jdk/jdk-10.0.2/bin:$PATH
 #export JAVA_OPTS="-XX:+IgnoreUnrecognizedVMOptions --add-modules java.se.ee"
 
-if [ ! -e ${ANDROID_SDK_INSTALLDIR}/build-tools/${ANDROID_BUILD_TOOLS_VERSION}/dx ]; then
+if [ ! -e ${ANDROID_SDK_INSTALLDIR}/build-tools/${ANDROID_BUILD_TOOLS_VERSION}/d8 ]; then
```

Comment:
New tools archive doesn't contain `dx` tool at all, so I keep this checking but use `d8` instead